plugins/pay.c

```diff
@@ -1269,6 +1269,9 @@ static struct pay_status *add_pay_status(struct pay_command *pc,
 	return ps;
 }
 
+#ifndef COMPAT_V090
+UNUSED
```

Comment:
this seems like a no-op? am i missing something?

Reply:
It basically just marks the function as `UNUSED` iff we are not in compat mode, to get the compiler's warning about unused functions to shut up :-)